vrs/marten-construction-script.py

```diff
@@ -1,168 +1,180 @@
-rand_path = 'gs://gnomad-tmp-4day/marten-temp-01-06/ht_50k_temp.ht'
-ht_50k = ht_50k.checkpoint(rand_path,overwrite = True)
-
-# 10,000 indels
-ht_10k_indel = ht_whole.filter(hl.is_indel(ht_whole.alleles[0], ht_whole.alleles[1]))
-# ht_whole.filter(ht_whole.vep.variant_class != "SNV")
-indel_sample_freq = 10000/(whole_count*indel_freq) # due to repeat samples, looking for 1.5x what we were expecting
-ht_10k_indel = ht_10k_indel.sample(indel_sample_freq).repartition(100)
-# TO ADD: ht_10k_indel = ht_10k_indel.repartition(100)
-indel_path = 'gs://gnomad-tmp-4day/marten-temp-01-06/ht_10k_indel_temp.ht'
-ht_10k_indel = ht_10k_indel.checkpoint(indel_path,overwrite = True)
-
-# 10,000 variants with long reference alleles
-ht_10k_longRef = ht_whole.filter(hl.len(ht_whole.alleles[0])>5)
-longRef_sample_freq = 10000/(whole_count*ref_freq)
-ht_10k_longRef = ht_10k_longRef.sample(longRef_sample_freq).repartition(10)
-# TO ADD: ht_10k_longRef = ht_10k_longRef.repartition(100)
-longRef_path = 'gs://gnomad-tmp-4day/marten-temp-01-06/ht_10k_longRef_temp.ht'
-ht_10k_longRef = ht_10k_longRef.checkpoint(longRef_path,overwrite = True)
-
-# 10,000 variants with long variant alleles
-ht_10k_longVar = ht_whole.filter(hl.len(ht_whole.alleles[1])>5)
-longVar_sample_freq = 10000/(whole_count*var_freq)
-ht_10k_longVar = ht_10k_longVar.sample(longVar_sample_freq).repartition(100)
-# TO ADD: ht_10k_longVar = ht_10k_longVar.repartition(100)
-longVar_path = 'gs://gnomad-tmp-4day/marten-temp-01-06/ht_10k_longVar_temp.ht'
-ht_10k_longVar = ht_10k_longVar.checkpoint(longVar_path,overwrite = True)
-
-# 9,000 X-Chromosome Variants
-ht_9k_x = ht_whole.filter(ht_whole.locus.contig == "chrX")
-x_sample_freq = 9000/(whole_count*x_freq)
-ht_9k_x = ht_9k_x.sample(x_sample_freq).repartition(100)
-# TO ADD: ht_9k_x = ht_9k_x.repartition(100)
-xchr_path = 'gs://gnomad-tmp-4day/marten-temp-01-06/ht_9k_x_temp.ht'
-ht_9k_x = ht_9k_x.checkpoint(xchr_path,overwrite = True)
-
-# 1,000 Y-Chromosome Variants 
-ht_1k_y = ht_whole.filter(ht_whole.locus.contig == "chrY")
-y_sample_freq = 1000/(whole_count*y_freq)
-ht_1k_y = ht_1k_y.sample(y_sample_freq)
-ht_1k_y = ht_1k_y.repartition(100)
-ychr_path = 'gs://gnomad-tmp-4day/marten-temp-01-06/ht_1k_y_temp.ht'
-ht_1k_y = ht_1k_y.checkpoint(ychr_path,overwrite = True)
-
-# Combine separate tables into one 
-
-tableParts = [ht_50k, ht_10k_indel, ht_10k_longRef, ht_10k_longVar, ht_9k_x, ht_1k_y]
-ht_union = tableParts[0]
-for item_to_union in tableParts[1:]:
-    ht_union = ht_union.union(item_to_union) 
-    # final table of all but the multiallelic sites
-    # at this point, approx ~90,000 variants
-
-# cannot figure how to make LOGGER work ? 
-print(f'Size of ~50k random, ~10k indel, ~10k long ref, ~10k long var, ~10k sex chr as: {ht_union.count()}')
+
+# Set a temporary path and variables to make writing checkpoints easier (and easier to edit for future users)
+tmp_path = 'gs://gnomad-tmp-4day/marten-temp-01-20-pr-v2/'
```

Comment:
```suggestion
tmp_path = "gs://gnomad-tmp-4day/marten-temp-01-20-pr-v2/"
```
this could also be 
```
tmp_path = args.tmp_path
```
and if you initialize this variable before your hail init, you could use 
```
tmp_dir=tmp_path
```
in the init